exercises/exercise-15/index.ts

```diff
@@ -1,6 +1,6 @@
 import {Database} from './database';
 import * as path from 'path';
-import {promises as fs} from 'mz/fs';
+import {promises as fs} from 'fs';
```

Comment:
I got runtime errors about `fs.copyFile` not being a function until I changed `mz/fs` → `fs`.